@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -704,6 +711,45 @@ export class TileGeometryCreator {
                     object.userData.geometryId = srcGeometry.uuid;
                 }
 
+                if (isFillTechnique(technique)) {
+                    object.onBeforeRender = chainCallbacks(
+                        object.onBeforeRender,
+                        (_renderer, _scene, _camera, _geometry, _material) => {
+                            if (_material.clippingPlanes === null) {
+                                _material.clippingPlanes = this.clippingPlanes;
+                                // TODO: Add clipping for Spherical projection.
+                            }
+                            if (mapView.projection.type === ProjectionType.Planar) {
+                                const expandFactor = 1.001;
```

Comment:
Why do you need to expand the bounding box?

Reply:
see: 1757b05049ccfe6ed659827f753da5b9901b43a4